superset-frontend/src/SqlLab/main.less

```diff
@@ -374,6 +374,24 @@ div.tablePopover {
   grid-auto-flow: column;
   grid-gap: 4px;
   padding-right: 8px;
+
+  .ReturnedRows {
+    margin-top: 48px;
+    margin-bottom: -32px;
+    font-size: 13px;
+    line-height: 24px;
+    span {
+      margin-left: 8px;
+    }
+  }
+  
+  .ReturnedRowsImage {
+    color: #FF7F43;
+    vertical-align: bottom;
+  }
+  .LimitMessage{
+    color: #8E94B0;
+  }
```

Comment:
we're going clean out these less files soon. I'd suggest using the emotion styles instead.

Reply:
@eschutho changed it to emotion, and also used your updated logic.